### ISSUE TITLE
Logging?

### ISSUE THREAD

**Issue by tricktrap** [RAR1741/RA19_RobotCode#69]
Logging was found to be unacceptably slow (between 40-70 milliseconds per iteration) just before a district event, so we took logging out.
- [x] The camera issue was fixed
- [x] The robot handling issue was fixed
- [x] We won our first championship in 14 years

Obviously I want to re-introduce logging.

**Comment by dracco1993** [comment]
Yeah, taking out logging let everything else controls wise go very well, but logging would've also let us figure out what was causing some of our issues faster.   🙏
